wallet/src/main/scala/org/bitcoins/wallet/api/SpvWalletApi.scala

```diff
@@ -0,0 +1,26 @@
+package org.bitcoins.wallet.api
+
+import org.bitcoins.core.bloom.BloomFilter
+
+import scala.concurrent.Future
+
+/**
+  * API for the wallet project.
+  *
+  * This wallet API is BIP44 compliant.
+  *
+  * @see [[https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki BIP44]]
+  */
+trait SpvWalletApi extends WalletApi {
```

Comment:
Same is true here, I believe. Also perhaps we want to move these into the core project eventually too? :/